libs/components/src/next/button/stories/vl-button.stories-arg.ts

```diff
@@ -175,6 +175,26 @@ export const buttonArgTypes: ArgTypes<ButtonArgs> = {
             defaultValue: { summary: buttonArgs.controlled },
         },
     },
+    ctaLink: {
+        name: 'cta-link',
+        description:
+            'Stelt de url in naar waar de link naar navigeert. Dit zorgt er ook voor dat de component een link (`<a>`) wordt met button-styling.<br>Zie documentatie voor meer info.',
```

Comment:
als je een acronym gebruikt, wel vermelden waar het voor staat, 2 keer naar is wel heel naar

Reply:
'Stelt de url in waar de hyperlink naar navigeert.<br>Instellen van call to action link attribuut, zorgt er ook voor dat de component een link (`<a>`) wordt met button-styling.<br>Zie documentatie voor meer info.',
?